README.md

```diff
@@ -1,13 +1,16 @@
 # Google Student Training in Engineering Program
 
-This repo contains the projects you'll work on for the first half of STEP.
-This work is based on the [Google Software Product Sprint](https://g.co/softwareproductsprint) program.
+## Overview
 
-To get started:
+This repo contains Bogdan Ciobanu's portfolio and STEP projects.
 
-- Login to [Google Cloud Shell](https://ssh.cloud.google.com/cloudshell/editor)
-- Clone this repo: `cd; git clone https://github.com/googleinterns/step.git`
-- Open the GitHub setup walkthrough: `teachme ~/step/walkthroughs/week-1-setup/github-setup-walkthrough.md`
-  - If the tutorial panel does not open or display the walkthrough contents, try refreshing the page.
+## Contents
 
-Then follow the on-screen instructions to set up your repo.
+- A web app that contains the bio and portfolio of the author
+- Walkthroughs for building a similar app
+
+## How to use
+
+    Run the shell command `mvn package appengine:run` from the portfolio
+directory or `./preview.sh` from the project's root. The webpage will be
```

Comment:
(correct me if I'm wrong): `./preview.sh` does not exist (yet). Consider updating this README with this part in the pull request that adds it.

Reply:
you are right, but won't that create a merge conflict?

Reply:
Yes, this may lead to a merge conflict with the other PRs as-is. I think this is a great opportunity to explore how to resolve merge conflicts in git/github. This will be a handy skill to develop for later when we work collaboratively, possibly touching the same files in the shared project.
I suggest https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/resolving-a-merge-conflict-on-github as a reference.